_source/api/logzio-public-api.yml

```diff
@@ -103,6 +102,10 @@ tags:
   - name: Manage shared tokens
     description: >-
       **Note:** Shared token endpoints require permissions that must be set by our Support team. Please email help@logz.io for assistance.
+  - name: Manage log shipping tokens
+    description: >-
+      Use these API requests to create, update, retrieve, or delete shipping tokens.
+      Data shipped using this token will be indexed under the account associated with this ID.
```

Comment:
@shalper "Data shipped using this token" - In this case, we're not referring to a specific token.